src/azure-cli/azure/cli/command_modules/rdbms/randomname/adjectives.txt

```diff
@@ -0,0 +1,166 @@
+aboard
```

Comment:
Also need to add `adjectives.txt` and `nouns.txt` in `setup.py`, otherwise these 2 files will not include the final distribution.

Reply:
Added